conan/recipes/beman/all/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+cmake_minimum_required(VERSION 3.15)
+project(beman NONE)
```

Comment:
project(beman VERSION 0.0.1
                         DESCRIPTION Beman project libraries
                         HOMEPAGE_URL https://github.com/beman-project
                         LANGUAGES C++)

Reply:
`LANGUAGES CXX`, not `C++`.

Reply:
See https://github.com/beman-project/beman/pull/21#discussion_r1623229936 but I expect this CMakeLists.txt to go away.

Reply:
I've removed this file.